### ISSUE TITLE
Fragment.Image.View alt text missing

### ISSUE THREAD

**Issue by ghost** [prismicio/scala-kit#14]
According to the API docs, the following are defined on the Image:

asHtml, getView, main, views

And in a View the following are defined:

asHtml, height, width, ratio, url.

There does not appear to be a way to access the Alt text as entered in the Writing Room
